src/aiida/orm/nodes/data/remote/base.py

```diff
@@ -185,3 +194,152 @@ def _validate(self):
 
     def get_authinfo(self):
         return AuthInfo.get_collection(self.backend).get(dbcomputer=self.computer, aiidauser=self.user)
+
+    def get_size_on_disk(
+        self,
+        relpath: Path | None = None,
+        method: str = 'du',
+        return_bytes: bool = False,
+    ) -> int | str:
+        """Connects to the remote Computer of the `RemoteData` object and returns the total size of a file or a
+        directory at the given `relpath` in a human-readable format.
+
+        :param relpath: File or directory path for which the total size should be returned, relative to
+            ``self.get_remote_path()``.
+        :param method: Method to be used to evaluate the directory/file size (either ``du`` or ``stat``).
```

Comment:
I changed it because I think people are more familiar with `stat`, as it is also a Linux command line utility, whereas `lstat` is not. Also, `lstat` is just the same as `os.stat(follow_symlinks=False)`, and I don't think the implementation detail should make the user-facing option less understandable. Lastly, I also think it's self-explanatory that, to get the size of a directory on disk, symlinks that point to files/directories that life somewhere else, should not be considered. That being said, we could also think about exposing the `follow_symlinks` option to the user, though that would also mean updating the implementation of `listdir_withattributes` of the `Transport`, so I wouldn't do that now, this PR has been dragging on for way too long anyway :D